### ISSUE TITLE
Ember 2.13 beta no longer supports router.router

### ISSUE THREAD

**Issue by danielspaniel** [DockYard/ember-route-action-helper#54]
You probably  already know this, but if not: 

 Currently this: 

```javascript 
 function getRoutes(router) {
    return emberArray(router.router.currentHandlerInfos).mapBy('handler').reverse();
  }
```

I think should change to this:
 
```javascript 
 function getRoutes(router) {
    return emberArray(router._routerMicrolib.currentHandlerInfos).mapBy('handler').reverse();
  }
```

I know it is still ember 2.13 beta ... so it's no rush or anything, but if you want me to submit a PR then let me know.  Might be something else that affects this .. not sure.

**Comment by urbany** [comment]
Ember 2.13 was just released. Seems like there is already an PR to fix this #53

**Comment by poteto** [comment]
Released in `2.0.3`
